task_11/index.html

```diff
@@ -0,0 +1,351 @@
+			    				</span>
+			    			</label>
+						</div>
+						<div class="payment-details">
+							<div class="payment-details-card">
+								<p>Card number</p>
+								<input required type="text" autocomplete="cc-number" id="user-card-number" name="user-card-number" pattern="[0-9]{16,16}" minlength="16" maxlength="16">
+								<label for="user-card-number"></label>
+							</div>
+							<div class="payment-details-date">
+								<p>Expiry date</p>
+								<input required type="month" autocomplete="cc-exp" id="user-card-date" name="user-card-date" pattern="[0-9]{2}/[0-9]{2}" placeholder="MM/YY">
+								<label for="user-card-date"></label>
+							</div>
+							<div class="payment-details-code">
+								<p>Code</p>
+								<input required type="text" autocomplete="cc-csc" id="user-card-code" name="user-card-code" pattern="[0-9]{3,3}" minlength="3" maxlength="3">
+								<label for="user-card-code"></label>
+							</div>
+						</div>
+					</div>
+				</fieldset>
+				<!-- End payment subsection -->
+				<!-- Begin submit subsection -->
+				<fieldset class="order-form-container">
+					<div class="submit-wrapper">
+						<div class="submit-terms">
+						<input required type="checkbox" name="terms" value="agree" id="terms">
+						<label for="terms">
+							<p>I agree to the <a href="">Terms &amp; Conditions</a></p>
+						</label>
+						</div>
+						<div class="summary-buttons">
+							<button type="submit" name="submit">Place order</button>
+							<button type="button" name="quote">Print a quote</button>
+						</div>
+					</div>
+					
+				</fieldset>
+				<!-- End submit subsection -->
+			</form>
+		</section>
+		<!-- End main order form section -->
+	</section>
+	<!-- End main section -->
+	<!-- Begin footer -->
+	<footer class="footer">
+		<div class="footer-container">
+			<div class="footer-item">
+				<div>
+					<i class="far fa-heart"></i>
+				</div>
+				<div>
+					<h3>What we do</h3>
+					<p>We provide a full range of language services including content development, document, website and social media accounts translation</p>
+				</div>
+			</div>
+			<div class="footer-item">
+				<div>
+					<i class="far fa-compass"></i>
+				</div>
+				<div>
+					<h3>Our main office</h3>
+				<address>
+					Dubai, UAE<br>
+					P.O. Box: 502735<br>
+					<a href="tel:+971044278128">Phone: +971(0)44278128</a>
+				</address>
+				</div>
+			</div>
+			<div class="footer-item">
+				<div>
+					<i class="fas fa-share-alt"></i>
+				</div>
+				<div>
+					<h3>Connect with us</h3>
+				<address>
+					<a href=""><i class="fab fa-facebook-f"></i>Facebook</a><br>
+					<a href=""><i class="fab fa-twitter"></i>Twitter</a><br>
+					<a href=""><i class="fab fa-linkedin-in"></i>Linkedin</a>			
+				</address>
+				</div>
+			</div>
+			<div class="footer-item">
+				<img src="images/logo_footer.png" class="logo" alt="logo">
+			</div>
+		</div>
+	</footer>
+	<!-- End footer -->
+</body>
+</html>
```

Comment:
Dont forget to add new line at the and of each doc or add this option to you editor.

Reply:
Done)